install_homebrew.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/sh
+
+# A simple Homebrew installation wrapper, to hide the platform-specific stanzas:
+# macOS: https://brew.sh/
+# Linux: https://docs.brew.sh/Homebrew-on-Linux#install
+
+set -e
+umask 022
```

Comment:
This is disabling group-write access (relative to the default, which I think is 022) for files created by this script - what's the need for this?

Reply:
Copy-n-paste-kid! I think I had taken [this](https://github.com/Linuxbrew/install/blob/master/install.sh#L4) as a base script.